sdk/search/Azure.Search.Documents/tests/DocumentOperations/IndexingTests.cs

```diff
@@ -53,6 +53,79 @@ public IndexingTests(bool async, SearchClientOptions.ServiceVersion serviceVersi
         }
         #endregion Utilities
 
+        [Test]
+        public async Task IndexingConveniences()
+        {
+            await using SearchResources resources = await SearchResources.CreateWithEmptyHotelsIndexAsync(this);
+            SearchClient client = resources.GetSearchClient();
+
+            // Upload
+            var doc1 = new SearchDocument
+            {
+                ["hotelId"] = "1",
+                ["hotelName"] = "Highway Hole in the Wall"
+            };
+            var doc2 = new SearchDocument
+            {
+                ["hotelId"] = "2",
+                ["hotelName"] = "Freeway Flophouse"
```

Comment:
Tip: don't ask from where they source their sausage for their continental breakfast; also, don't eat their continental breakfast.